### PR TITLE
Camo Everything

### PR DESCRIPTION
**Pull request by josh**
Camo's all images including `https`.

Removed `http/https://github.com` image rewriting. Those shouldn't but linked to in GFM. So no one ought to be doing that. They should be using github helpers so they point to our CDN. Also, this would point them to a CDN through camo. Fewer edge cases.

/cc @jch @atmos

### CODE REVIEW

File: test/html/pipeline/camo_filter_test.rb [hunk 1]
```diff
@@ -36,10 +30,10 @@ def test_handling_images_with_no_src_attribute
       CamoFilter.call(orig, @options).to_s
     end
   end
-    
+
   def test_required_context_validation
-    exception = assert_raise(ArgumentError) { 
-      CamoFilter.call("", {}) 
+    exception = assert_raise(ArgumentError) {
+      CamoFilter.call("", {})
```
Comment (zanuda777) [inline test/html/pipeline/camo_filter_test.rb]: -

**Comment by Ibocann** [comment]
image
